app/models/workarea/checkout.decorator

```diff
@@ -1,5 +1,12 @@
 module Workarea
   decorate Checkout, with: :kount do
+    # User-defined fields for Kount.
+    #
+    # @return [Hash]
+    def kount_request_user_defined_fields
+      {}
```

Comment:
This was something carried over from the original implementation. I'm not sure what it's used for, couldn't find it actually getting set anywhere.

Reply:
So why not delete it?